test/shared-class.test.js

```diff
@@ -295,6 +295,70 @@ describe('SharedClass', function() {
       expect(methods).to.not.contain(INST_METHOD_NAME);
       expect(methods).to.not.contain('instTestMethod');
     });
+
+    it('excludes disabled prototype methods from the method list using ' +
```

Comment:
Please reword this and others as:
`excludes prototype methods from method list disabled by alias`